src/main/java/games/strategy/triplea/ai/weakAI/WeakAI.java

```diff
@@ -1064,6 +1064,4 @@ private static void doPlace(final Territory where, final Collection<Unit> toPlac
   public boolean shouldBomberBomb(final Territory territory) {
     return true;
   }
-
-  public static final Predicate<Unit> Transporting = o -> TripleAUnit.get(o).getTransporting().size() > 0;
```

Comment:
Instead of renaming this field, I just replaced its usage with `Matches#unitIsTransporting()`, which seems to be equivalent:

```java
public static Predicate<Unit> unitIsTransporting() {
  return unit -> {
    final Collection<Unit> transporting = TripleAUnit.get(unit).getTransporting();
    return !(transporting == null || transporting.isEmpty());
  };
}
```